src/osd/OSD.cc

```diff
@@ -5942,7 +5943,8 @@ OSDService::ScrubJob::ScrubJob(const spg_t& pg, const utime_t& timestamp, bool m
       sched_time += rand() % (int)(g_conf->osd_scrub_min_interval *
```

Comment:
Changing both min/max with the pool setting you'd have to document something like "Setting the pool_scrub_interval will cause the pool to scrub at the interval selected.  The load average will no longer be considered when scheduling the pool's scrubs."

I don't think this is the way to go since the delay after scrub_min_interval due to load may be vital to preventing scrubbing from impacting client activity.